databases/Assignment2_answers.sql

```diff
@@ -0,0 +1,316 @@
+1. Which laptop models are shown in the Product relation?
+SELECT DISTINCT(Model)
+FROM Product
+WHERE Type = laptop;
+
+2. Find the set of all manufacturers ordered alphabetically.
+SELECT DISTINCT(Manufacturer)
+FROM Product
+ORDER BY DESC; # is there an alpha order?
+# all data types are by default order-able
+
+
+3. Which computer models appear in the database?
+SELECT Model
+FROM Product
+WHERE Type = pc OR Type = laptop;
+
+OR --> (below is a better idea lol)
+
+SELECT DISTINCT(Model)
+FROM Computer;
+
+
+4. Which computer models use 256 MB of RAM?
+SELECT Model
+FROM Computer
+WHERE Computer.RAM = 256;
```

Comment:
should i expect the units to be part of the data entry?
ie where I would have to do something like:
WHERE Computer.RAM = '256MB' 
since adding the units would make the data_type = string